--- a/cppuhelper/source/servicemanager.cxx	2022-03-02 16:49:44.000000000 +0100
+++ b/cppuhelper/source/servicemanager.cxx	2022-03-04 23:13:24.510649953 +0100
@@ -1900,6 +1900,12 @@
                 static_cast< cppu::OWeakObject * >(this));
         }
 
+        if (rEntry.second->prefix == "rptui" ||
+                rEntry.second->prefix == "rptxml" ||
+                rEntry.second->>prefix == "rpt") {
+            continue;
+        }
+
         // load component library
         osl::Module aModule(aUri, SAL_LOADMODULE_NOW | SAL_LOADMODULE_GLOBAL);
 
